demisto_sdk/commands/validate/sdk_validation_config.toml

```diff
@@ -158,6 +158,7 @@ select = [
   "BA118",
   "BA119",
   "BA126",
+  "BA128",
```

Comment:
should also put it under 'use_git' section, we want to validate this during pr builds & nightlies.

Reply:
Added under `[use_git] select` in commit d49bff905078cc4b9794bed4d2d6d59589faf278